Makefile

```diff
@@ -1,20 +1,37 @@
 PREFIX ?= /usr/local
-MAN_PREFIX ?= ${PREFIX}/man
+BINDIR ?= $(PREFIX)/bin
+LIBDIR ?= $(PREFIX)/lib
+MANDIR ?= $(PREFIX)/man
+SHAREDIR ?= $(PREFIX)/share
+EXAMPLESDIR ?= $(SHAREDIR)/examples
+
+.PHONY: all install test distrib
 
 all: target/release/pizauth
 
 target/release/pizauth:
 	cargo build --release
 
+PLATFORM=$(shell uname)
+
 install: target/release/pizauth
-	install -d ${PREFIX}/bin
-	install -c -m 555 target/release/pizauth ${PREFIX}/bin/pizauth
-	install -d ${MAN_PREFIX}/man1
-	install -d ${MAN_PREFIX}/man5
-	install -c -m 444 pizauth.1 ${MAN_PREFIX}/man1/pizauth.1
-	install -c -m 444 pizauth.conf.5 ${MAN_PREFIX}/man5/pizauth.conf.5
-	install -d ${PREFIX}/share/examples/pizauth
-	install -c -m 444 pizauth.conf.example ${PREFIX}/share/examples/pizauth
+	install -d ${DESTDIR}${BINDIR}
+	install -c -m 555 target/release/pizauth ${DESTDIR}${BINDIR}/pizauth
```

Comment:
Til this is a BSDism -- both the separation of `-d` away from the installation of files, and the use of `-c` make little sense under linux, where `-D` (combining `install -d $dir` and `install $file $dir`) exists and `-c` is ignored.
https://man.archlinux.org/man/install.1
https://man.freebsd.org/cgi/man.cgi?query=install&sektion=1

Nothing to change, just a TIL